src/write/unit.rs

```diff
@@ -545,6 +545,15 @@ impl DebuggingInformationEntry {
         self.children.iter()
     }
 
+    /// Delete a child entry and all of its children.
+    ///
+    /// # Panics
+    ///
+    /// Panics if `id` is invalid.
```

Comment:
I don't think this panics.

Reply:
Oh that's right, I forgot to update the comment after changing the implementation. I'll fix it.